assets/base/i18n/en.json

```
@@ -995,6 +995,10 @@
   "playbooks.checklist_item.run_command": "Run command",
   "playbooks.checklist_item.skip": "Skip",
   "playbooks.checklist_item.skipped": "Skipped",
+  "playbooks.edit_command.label": "Command",
+  "playbooks.edit_command.placeholder": "Insert command here",
```

nitpick:
```suggestion
  "playbooks.edit_command.placeholder": "type a command here",
```

unless this specific text comes from UX